### PR TITLE
Document `satisfies`

### PR DESCRIPTION
**Pull request by RyanCavanaugh**
ref https://github.com/microsoft/TypeScript/issues/59930

### CODE REVIEW

File: packages/documentation/copy/en/handbook-v1/Basic Types.md [hunk 1]
```diff
@@ -384,7 +384,48 @@ let strLength: number = (<string>someValue).length;
 ```
 
 The two samples are equivalent.
-Using one over the other is mostly a choice of preference; however, when using TypeScript with JSX, only `as`-style assertions are allowed.
+Using one over the other is mostly a choice of preference; however, when using TypeScript with JSX, only `as` assertions are allowed.
+
+## `satisfies` checks
+
+While the `as` operator *asserts* that an expression is of a given type, the `satisfies` operator *checks` the type instead.
```
Comment (jakebailey) [inline packages/documentation/copy/en/handbook-v1/Basic Types.md:391]:
```suggestion
While the `as` operator *asserts* that an expression is of a given type, the `satisfies` operator *checks* the type instead.
```

**Comment by jakebailey** [comment]
This is the v1 handbook; is there a v2 place where this is documented?

**Comment by RyanCavanaugh** [comment]
StarTrekPatrickStewartGIF

**Comment by devonhk** [comment]
Will the `satisfies` operator ever be documented in the reference section? It's only mentioned in old release [notes](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html).

**Pull request closed without merging**